[2023-10-05T15:42:52.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Reddit_starbucks_Dag.extract_post_task manual__2023-10-05T15:42:49.333437+00:00 [queued]>
[2023-10-05T15:42:52.236+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Reddit_starbucks_Dag.extract_post_task manual__2023-10-05T15:42:49.333437+00:00 [queued]>
[2023-10-05T15:42:52.237+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-10-05T15:42:52.256+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_post_task> on 2023-10-05 15:42:49.333437+00:00
[2023-10-05T15:42:52.267+0000] {standard_task_runner.py:57} INFO - Started process 514 to run task
[2023-10-05T15:42:52.273+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Reddit_starbucks_Dag', 'extract_post_task', 'manual__2023-10-05T15:42:49.333437+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/reddit_starbucks_dag.py', '--cfg-path', '/tmp/tmp7xt4khjg']
[2023-10-05T15:42:52.279+0000] {standard_task_runner.py:85} INFO - Job 118: Subtask extract_post_task
[2023-10-05T15:42:52.354+0000] {task_command.py:415} INFO - Running <TaskInstance: Reddit_starbucks_Dag.extract_post_task manual__2023-10-05T15:42:49.333437+00:00 [running]> on host 83a26c877c38
[2023-10-05T15:42:52.463+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Reddit_starbucks_Dag' AIRFLOW_CTX_TASK_ID='extract_post_task' AIRFLOW_CTX_EXECUTION_DATE='2023-10-05T15:42:49.333437+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-05T15:42:49.333437+00:00'
[2023-10-05T15:42:54.768+0000] {logging_mixin.py:151} INFO - An error occurred in initial_data_transformation: If using all scalar values, you must pass an index
[2023-10-05T15:42:54.772+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/plugins/reddit_data_extractor.py", line 161, in extract_reddit_starbucks_details
    raise e
  File "/opt/airflow/plugins/reddit_data_extractor.py", line 147, in extract_reddit_starbucks_details
    posts_df = pd.DataFrame(details)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/frame.py", line 709, in __init__
    mgr = dict_to_mgr(data, index, columns, dtype=dtype, copy=copy, typ=manager)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/construction.py", line 481, in dict_to_mgr
    return arrays_to_mgr(arrays, columns, index, dtype=dtype, typ=typ, consolidate=copy)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/construction.py", line 115, in arrays_to_mgr
    index = _extract_index(arrays)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/internals/construction.py", line 645, in _extract_index
    raise ValueError("If using all scalar values, you must pass an index")
ValueError: If using all scalar values, you must pass an index
[2023-10-05T15:42:54.793+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Reddit_starbucks_Dag, task_id=extract_post_task, execution_date=20231005T154249, start_date=20231005T154252, end_date=20231005T154254
[2023-10-05T15:42:54.812+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 118 for task extract_post_task (If using all scalar values, you must pass an index; 514)
[2023-10-05T15:42:54.853+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-05T15:42:54.887+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
